assets/images/Svg.js

```diff
@@ -49,6 +52,8 @@ module.exports = function(name, props) {
   if (name === "subSettings") {
     className = "";
   }
+
+  name = supportedLibraries.find(lib=>path && path.includes(lib)) || name;
```

Comment:
We should be a bit stricter here because we only want the webpack domain path to match and not other paths